todo_reporter/lib/src/todo.dart

```diff
@@ -1,6 +1,6 @@
 class Todo {
-  final String name;
-  final String todoUrl;
+  final String? name;
```

Comment:
This can't be nullable, since there's an assert in the constructor checking that it is not null.

I think this could be written like this:

```dart
class Todo {
  final String name;
  final String? todoUrl;

  const Todo(this.name, {this.todoUrl});
}
```

Thoughts?

Reply:
Yes, that's correct